src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -169,7 +168,7 @@ class SessionService(
         )
       )
     ).count {
-      it.status == VisitStatus.BOOKED || it.status == VisitStatus.RESERVED
+      it.visitStatus == VisitStatus.BOOKED || it.visitStatus == VisitStatus.RESERVED
```

Comment:
maybe reduce duplication (see line 141) of code and make a method call isBookedOrReserved, may something more like isCountStatus(it) your call

Reply:
Ill see if i can improve